scorecard/Makefile

```diff
@@ -0,0 +1,17 @@
+build:
```

Comment:
In particular, make can't tell if the Dockerfile has changed because the docker image is stored in local docker storage (no timestamp).  We could make, say, the docker build target non-phony and create (touch) a build stamp file, but that seems error prone since it isn't the actual build target.  Since docker does build caching, I prefer to rely on that.

Reply:
Point. Did some reading as to how docker build works. Objections withdrawn.